pkg/skaffold/survey/survey.go

```diff
@@ -46,21 +47,36 @@ var (
 	isStdOut             = output.IsStdout
 	open                 = browser.OpenURL
 	updateSurveyPrompted = sConfig.UpdateGlobalSurveyPrompted
+	parseConfig          = schema.ParseConfigAndUpgrade
 )
 
 type Runner struct {
-	configFile string
+	configFile     string
+	skaffoldConfig string
+	mode           sConfig.RunMode
+	surveyID       string
 }
 
-func New(configFile string) *Runner {
+func New(configFile string, skaffoldConfig string, mode string) *Runner {
```

Comment:
It would be nice if we could avoid reloading the configs for the survey and in the runner.  Perhaps we could have provider / loader that is passed here and into the runner so that we only load it once.  This can wait for later.